src/tweet.ts

```diff
@@ -37,18 +40,18 @@ export const makeStatus = (data: NewChallengeTweetData): string =>
 📣「${truncateText(100)(data.reasonGiven)}」
 
 👤 View the profile: ${data.pohProfileUrl}
-🔎 Follow the case: ${data.klerosCaseUrl}`
-
-
+🔎 Follow the case: ${data.klerosCaseUrl}
+#proofofhumanity`
```

Comment:
This will get us closer to the character limit, so I think we sure ensure we never go over it.

For reference, from playing around with the tweet editor:
- The character limit is 280
- URLs count for 23 characters independent of their actual length
- The hashtags count as whatever their length is, they don't seem to work like URLs
- Newlines count as 1
- Surprisingly, emojis count as 2

The challenge reason is already capped to 100 and seems to be enough to get the idea, judging by the tweet history.

The other variable is the name, I don't know if Proof of Humanity has a limit for the display name, but we could limit that, 30+ characters names are rare but very possible in spanish for example.

A simple cap for the name of whatever remains after the rest of the tweet should be good. We can balance the name and reason caps some other time.